src/common/Cargo.toml

```diff
@@ -49,6 +49,7 @@ hyper = "0.14"
 hytra = { workspace = true }
 itertools = "0.11"
 itoa = "1.0"
+jsonbb = { git = "https://github.com/risingwavelabs/jsonbb.git", rev = "45f7b53" }
```

Comment:
Do you plan to create a release version?

Reply:
Yes, I'll release it once all tests pass.